src/java.ts

```diff
@@ -18,18 +18,17 @@
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 
-import fs from 'fs';
-import * as fsExtra from 'fs-extra';
 import AdmZip from 'adm-zip';
-import zlib from 'zlib';
-import path from 'path';
 import axios from 'axios';
 import crypto from 'crypto';
+import fs from 'fs';
+import * as fsExtra from 'fs-extra';
+import path from 'path';
+import semver, { SemVer } from 'semver';
 import * as stream from 'stream';
 import tarStream from 'tar-stream';
 import { promisify } from 'util';
-import semver, { SemVer } from 'semver';
-import { log, LogLevel } from './logging';
+import zlib from 'zlib';
```

Comment:
not for this PR, but lets align on a formatting ruleset we can enforce in the CI / locally to reduce unrelated changes (I like your setting to order alphabetically)
I created a subtask for this
https://sonarsource.atlassian.net/browse/SCANNPM-16